CMakeLists.txt

```diff
@@ -75,6 +72,7 @@ set(exported_libraries "")
 
 ## Put libbarrett headers in the include path
 include_directories("${PROJECT_SOURCE_DIR}/include/")
+include_directories("${PROJECT_BINARY_DIR}/include")
```

Comment:
In some linux commands, trailing slash acts on the sub folders while without it acts on the folder itself. I assume that this change includes the include directory itself and all its sub directories while the previous one includes just the sub folders. I am not sure about how important this is since I did not do this. However, I found on the internet that few projects that uses cmake had their include directories listed this way.